app/src/main/java/ru/practicum/android/diploma/favourites/data/db/Converters.kt

```diff
@@ -0,0 +1,65 @@
+package ru.practicum.android.diploma.favourites.data.db
+
+import ru.practicum.android.diploma.search.domain.models.Vacancy
+import ru.practicum.android.diploma.vacancy.domain.models.VacancyFull
+
+fun VacancyEntity.toVacancy(): Vacancy {
+    return Vacancy(
+        id = id,
+        name = name,
+        company = company,
+        currency = currency,
+        salaryFrom = salaryFrom,
+        salaryTo = salaryTo,
+        area = area,
+        icon = icon
+    )
+}
```

Comment:
Подобные мапперы можно писать немного короче
```suggestion
fun VacancyEntity.toVacancy(): Vacancy =
    Vacancy(
        id = id,
        name = name,
        company = company,
        currency = currency,
        salaryFrom = salaryFrom,
        salaryTo = salaryTo,
        area = area,
        icon = icon
    )
```